test/app/item_test.exs

```diff
@@ -141,7 +141,12 @@ defmodule App.ItemTest do
       started = NaiveDateTime.utc_now()
 
       {:ok, timer1} =
-        Timer.start(%{item_id: item1.id, person_id: 1, start: started})
+        Timer.start(%{
+          item_id: item1.id,
+          person_id: 1,
+          start: started,
```

Comment:
Should having the `timer.start` and `timer.start` the same return an `:error_start_equal_stop` instead of `{:ok, timer}` now? or am I reading the code incorrectly?